foreman/data_refinery_foreman/surveyor/harmony.py

```diff
@@ -253,6 +424,9 @@
 
 logger = get_and_configure_logger(__name__)
 
+# re.Pattern / re._pattern_type were not defined in 3.8?
+RE_TYPE = re.compile("").__class__
+
```

Comment:
It seems that at least `re.Pattern` was.

Reply:
Yeah it is in the docs, when I run the tests it is undefined. I am going to leave this as is for now and when the functionality is proven then we can come back and revisit this. I think it might be that our python version in the tests is different or something.